src/main/java/racingcar/domain/TryCount.java

```diff
@@ -0,0 +1,21 @@
+package racingcar.domain;
+
+public class TryCount {
+    private static final int TRY_COUNT_MIN = 0;
+    private final int tryCount;
```

Comment:
보통 상수와 객체의 필드는 개행으로 분리시켜 모아두는것이 가독성 측면에서 좋다고 생각해요!

```suggestion
    private static final int A = 0;
    private static final int B = 0;
    private static final int C = 0;
    
    private final int value1;
    private final int value2;
    private final int value3;
```

Reply:
전체적인 클래스를 상수와 객체의 필드를 개행으로 분리하는 방향으로 리팩토링해보았습니다 !